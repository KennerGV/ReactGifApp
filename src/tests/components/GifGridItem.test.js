import React from 'react';
const { shallow } = require("enzyme")
import {GifGridItem} from '../../components/GifGridItem'

describe('Pruebas en Gifgriditem', () => {
    
    const tittle = 'Un titulo';
    const url = 'https://localhost/algo.png';

    test('Debe de mostrar el componente correctamente', () => {
        
        const wrapper = shallow(<GifGridItem title={tittle} url={url}/>)
        expect(wrapper).toMatchSnapshot();

    })
    
})
