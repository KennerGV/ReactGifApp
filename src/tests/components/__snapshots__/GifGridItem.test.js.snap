// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en Gifgriditem Debe de mostrar el componente correctamente 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img
    alt="Un titulo"
    src="https://localhost/algo.png"
  />
  <p>
     
    Un titulo
     
  </p>
</div>
`;
